app/views/assets/view.php

```diff
@@ -0,0 +1,150 @@
+        </div>
+
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+            </ul>
+
+            <ul class="nav navbar-nav navbar-right">
+                <!--li><a href="#">Link</a></li-->
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-edit"></i>
+                        Sign Up
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-log-in"></i>
+                        Sign In
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-info-sign"></i>
+                        Help
+                    </a>
+                </li>
+            </ul>
+
+            <ul class="nav navbar-nav navbar-left">
+                <li>
+                    <a href="#menu-toggle" id="menu-toggle">
+                        Modules
+                    </a>
+                </li>
+            </ul>
+
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
+<body style="padding: 50px">
+<div id="wrapper" class="toggled">
+    <!-- Sidebar -->
+    <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+
+            <li>
+                <a href="<?php echo base_url(); ?>missing/">
+                    Missing People
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>organisations/">
+                    Organisations
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Camps
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>assets/">
+                    Assets
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Requests
+                </a>
+            </li>
+
+        </ul>
+    </div>
+    <div id="page-content-wrapper" style="background-color: #ffffff;">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="page-header">
+                        <h1 style="text-align: center;">
+                            Assets Available<br/>
+                        </h1>
+                        <br /><br />
+                    </div>
+                    <?php
+                        foreach ($assets_list as $assets) {
+                    ?>
+                    <div class='panel panel-primary'>
+                        <div class='panel-heading' style='font-size: large; padding-bottom: 15px'>
+                            <?php echo "<b>Asset Name: " . $assets['name']."</b>"; ?>
```

Comment:
Since you did the changes I suggested, one more suggestion, 
Instead of
    `<?php echo "<b>Asset Name: " . $assets['name']."</b>"; ?>`
use
    `<b>Asset Name: <?php echo $assets['name'] ?></b>`